snooty/rstparser.py

```diff
@@ -365,6 +365,38 @@ def run(self) -> List[docutils.nodes.Node]:
         return [node]
 
 
+class VersionDirective(docutils.parsers.rst.Directive):
```

Comment:
I'm a little hypocritical since I'm not the best role model in this regard, but a docstring here would be good describing what the directive is, and why we need to override the normal directive handling behavior